ironfish/src/consensus/verifier.ts

```diff
@@ -296,16 +294,15 @@ export class Verifier {
   async hasValidSpends(block: Block, tx?: IDatabaseTransaction): Promise<VerificationResult> {
```

Comment:
The errornously removed code was intending to check for  `the note being spent really existed in the tree at the time it was spent` but was not. Should this still check for that some how? The spend size is the size of the nullifier tree when the commitment was created, not the note tree right?